Dockerfile

```diff
@@ -14,6 +14,12 @@ RUN curl -sL https://deb.nodesource.com/setup_16.x | bash - && \
     pip install --no-cache-dir -r requirements/dev-requirements.txt && \
     # install mlflow in editable form
     pip install --no-cache-dir -e . && \
+    groupadd -g 10001 mlflow && \
+    useradd -u 10001 -g 10001 -m -d /home/mlflow mlflow
+
+# Build MLflow UI
+RUN curl -sL https://deb.nodesource.com/setup_16.x | bash - && \
```

Comment:
Why the nodejs install here?

Reply:
IDK, that was already in the Dockerfile.. :)

Reply:
lol sorry it's late. Gotcha.